### PR TITLE
[Analyzer] Test and generated permissions for HuggingFace, Square & Stripe

### PR DESCRIPTION
**Pull request by abmussani**
### Description:
This PR includes test and generated enumerated permissions for HuggingFace, Square & Stripe Analyzers.

### Checklist:
* [ ] Tests passing (`make test-community`)?
* [x] Lint passing (`make lint` this requires [golangci-lint](https://golangci-lint.run/usage/install/#local-installation))?

### CODE REVIEW

File: pkg/analyzer/analyzers/square/permissions.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+permissions:
+  - BANK_ACCOUNTS_READ
+  - APPOINTMENTS_WRITE
+  - APPOINTMENTS_ALL_WRITE
+  - APPOINTMENTS_READ
+  - APPOINTMENTS_ALL_READ
+  - APPOINTMENTS_BUSINESS_SETTINGS_READ
```
Comment (mcastorina) [inline pkg/analyzer/analyzers/square/permissions.yaml:7]: Is there a reason why these are all capitalized? I don't recall any other analyzer doing that.

**Review by mcastorina — APPROVED** [review]
lgtm!

File: pkg/analyzer/analyzers/square/permissions.yaml (hunk 1 above)
Comment (abmussani) [reply pkg/analyzer/analyzers/square/permissions.yaml:7]: I think I copied it from somewhere. You are right, all of our analyzers have permissions in small caps. Let me change this one in the name of consistency.

Comment (abmussani) [reply pkg/analyzer/analyzers/square/permissions.yaml]: @mcastorina updated the permissions with small caps for consistency.

**Review by mcastorina — APPROVED** [review]

**Pull request merged**